### PR TITLE
chore: replace textarea with rc-textarea

### PR DESCRIPTION
**Pull request by kerm1it**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | - |
| 🇨🇳 Chinese | - |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/TextArea.tsx [hunk 1]
```diff
@@ -1,17 +1,13 @@
 import * as React from 'react';
+import RcTextArea, { TextAreaProps as RcTextAreaProps } from 'rc-textarea';
+import ResizableTextArea from 'rc-textarea/lib/ResizableTextArea';
```
Comment (afc163) [inline components/input/TextArea.tsx:3]: 为啥是 lib。

Comment (afc163) [inline components/input/TextArea.tsx:3]:
```suggestion
import ResizableTextArea from 'rc-textarea';
```

Comment (afc163) [reply components/input/TextArea.tsx:3]: ResizableTextArea 也从 `rc-textarea` 中引出来吧。

Comment (Kermit-Xuan) [reply components/input/TextArea.tsx:3]: 默认导出是 `RcTextArea`，`RcTextArea` 内部有 `ResizableTextArea`。

Comment (Kermit-Xuan) [reply components/input/TextArea.tsx:3]: 那建议是把两个合在一块写是吧？

Comment (afc163) [reply components/input/TextArea.tsx:3]: 统一从 index 里导出。

Comment (afc163) [reply components/input/TextArea.tsx:3]: 因为有 es 和 lib 两个入口，写死不太好。

Comment (Kermit-Xuan) [reply components/input/TextArea.tsx:3]: OK，那我改了后再发个minor版本吧。

Comment (Kermit-Xuan) [reply components/input/TextArea.tsx:3]: 之前在哪个库里也见到从 lib 导出好像

Comment (afc163) [reply components/input/TextArea.tsx:3]: rc-util 没有 index 入口，统一从 lib 引入。

File: components/input/__tests__/textarea.test.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { mount } from 'enzyme';
 // eslint-disable-next-line import/no-unresolved
+import RcTextArea from 'rc-textarea';
```
Comment (yoyo837) [inline components/input/__tests__/textarea.test.js:4]: // eslint-disable-next-line import/no-unresolved 
会错误的作用到新行

Comment (Kermit-Xuan) [reply components/input/__tests__/textarea.test.js:4]: OK, 我把这删了

**Review by xrkffgg — APPROVED** [review]

**Pull request merged**